code/game/gamemodes/changeling/powers/panacea.dm

```diff
@@ -10,6 +10,12 @@
 /obj/effect/proc_holder/changeling/panacea/sting_action(mob/user)
 	user << "<span class='notice'>We begin cleansing impurities from our form.</span>"
 
+	var/mob/living/simple_animal/borer/borer = user.has_brain_worms()
+	if(borer)
+		borer.leave_victim()
+		if(iscarbon(user))
+			var/mob/living/carbon/C = user
+			C.vomit(0)

```

Comment:
Make sure the borer can't do wonky stuff where if it uses anatomic while controlling it vomits itself out. Also maybe a message here so the borer and the user knows what happened when they use it.